app/models/card.rb

```diff
@@ -1,2 +1,18 @@
 class Card < ActiveRecord::Base
+  validates :original_text, :translated_text, :review_date, presence: { message: I18n.t('error.validation.messages.cant_be_blank') }
+
+  validate :check_difference
+
+  before_validation :set_review_date
+
+  private
+
+  def check_difference
+    errors.add( I18n.t('error.validation.description.the_same_value') ,I18n.t('error.validation.messages.the_same_value')) if
```

Comment:
Line is too long. [125/80]<br>Space missing after comma.<br>Space found before comma.<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.